run_benchmarks.rb

```diff
@@ -318,7 +318,7 @@ def run_benchmarks(ruby:, ruby_description:, name_filters:, out_path:, pre_init:
 # Create the output directory
 FileUtils.mkdir_p(args.out_path)
 
-metadata = {}
+metadata = { end_time: Time.now.strftime("%Y-%m-%d %H:%M:%S %Z (%z)") }
```

Comment:
See https://github.com/Shopify/yjit-bench/pull/136#discussion_r1027908435
This wouldn't capture the time at the right time.

But maybe this end_time should just be removed, cc @noahgibbs ?

Reply:
ugh yeah okay. But, given that Noah (who maintains yjit-metrics) is not sure if we should maintain this, I'm in favor of just removing it. Even when I was running this script multiple times in batches, I didn't utilize that. Just showing "Total time spent benchmarking" seems enough in most cases.

Reply:
removed it https://github.com/Shopify/yjit-bench/pull/139/commits/74e4cde1d5b1c31a88f1e023c72217299bd72d17

Reply:
Yeah, I'm fine with removing it.